.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!-- 
+Thank you so much for contributing a pull request! Please ensure
+that your pull request satisfies the checklist before submitting:
+https://www.acoular.org/contributing/checklist.html
+
+If you are part the of Acoular development team, please make sure to
+select a label for this pull request on the sidebar to the right.
+
+Again, thanks for contributing!
+-->
+
+### Description
+<!-- Please provide a detailed description of your changes. Include any relevant context or background. -->
+
+### Reference issue
+<!-- If this pull request closes a issue, please denote it like: Closes #141. -->
+
+### Milestone
+<!-- Please add the milestone you want this pull request to be included in.
+     See possible milestones here: https://github.com/acoular/acoular/milestones
+-->
+
+### Reviewer
+<!-- Please notify a reviewer by mentioning them below with their tag. Possible reviewers are:
```

Comment:
This list should include all members of the Acoular Development Team. If it is unclear to whom the PR should be assigned, it can be left blank. Then, a member of the Acoular Development Team will assign another member. 

We may consider using random assignment in the future: https://github.com/marketplace/actions/reviewer-lottery.